.github/workflows/release-notes.yml

```diff
@@ -0,0 +1,23 @@
+name: Release new version
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  create-release-notes:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+    steps:
+      - name: Create release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          tag: ${{ github.ref_name }}
```

Comment:
I think it's best practice to put these `${{ ... }}` expansions in quotes. idk whether this really matters.